test/functional/services/remote/webdriver.ts

```diff
@@ -86,6 +87,8 @@ async function attemptToCreateCommand(
           'use-fake-device-for-media-stream',
           // Bypass the media stream infobar by selecting the default device for media streams (e.g. WebRTC). Works with --use-fake-device-for-media-stream.
           'use-fake-ui-for-media-stream',
+          // Allow testing with self-signed certificates
+          'ignore-certificate-errors',
```

Comment:
Can we pass it as an optional argument for integration tests only?

Reply:
Ah, because this is used elsewhere?

Reply:
We had discussing before that we don't want to ignore any errors on regular CI runs